src/Interpreters/JoinedTables.cpp

```diff
@@ -239,7 +240,7 @@ bool JoinedTables::resolveTables()
     const auto & settings = context->getSettingsRef();
     bool include_alias_cols = include_all_columns || settings.asterisk_include_alias_columns;
     bool include_materialized_cols = include_all_columns || settings.asterisk_include_materialized_columns;
-    tables_with_columns = getDatabaseAndTablesWithColumns(table_expressions, context, include_alias_cols, include_materialized_cols);
+    tables_with_columns = getDatabaseAndTablesWithColumns(table_expressions, context, include_alias_cols, include_materialized_cols, is_create_parameterized_view);
```

Comment:
let's add a comment that we need to pass is_create_parameterized_view in order to make getSampleBlock work properly? as it is not obvious how it may be needed in method named getDatabaseAndTablesWithColumns

Reply:
addressed it